src/build_workflow/build_target.py

```diff
@@ -7,31 +7,34 @@
 import os
 import uuid
 
-from system.arch import current_arch
+from system.os import current_arch, current_platform
 
 
 class BuildTarget:
     build_id: str
     name: str
     version: str
+    platform: str
     arch: str
     snapshot: bool
     output_dir: str
 
     def __init__(
         self,
         version,
+        platform=None,
         arch=None,
         name=None,
         snapshot=True,
         build_id=None,
         output_dir="artifacts",
     ):
-        self.build_id = os.getenv("OPENSEARCH_BUILD_ID") or os.getenv("OPENSEARCH_DASHBOARDS_BUILD_ID") or build_id or uuid.uuid4().hex
+        self.build_id = os.getenv("BUILD_NUMBER") or build_id or uuid.uuid4().hex
```

Comment:
Side effect of https://github.com/opensearch-project/opensearch-build/pull/730 that we forgot to fix.

Reply:
Yea, looks like those bundle numbers became `8c363d37f71349b1add343c0c48945a7`